subsys/can/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+if(CONFIG_CAN_STACKS)
+  zephyr_interface_library_named(CAN_STACKS)
+
+  zephyr_library()
+  zephyr_library_sources(common.c)
```

Comment:
Why do you build a lib here? If it's compiled in, you can use the common helpers anyway.

Reply:
Ah, now i know what you'r doing.